includes/components/class-attribute.php

```diff
@@ -307,6 +310,72 @@ public function register_models() {
 		}
 	}
 
+	/**
+	 * Validates unique model attributes.
+	 *
+	 * @param array  $errors Error messages.
+	 * @param object $model Model object.
+	 * @return array
+	 */
+	public function validate_unique_attributes( $errors, $model ) {
+		if ( empty( $errors ) ) {
+
+			// Set unique fields.
+			$fields = [];
+
+			foreach ( $model->_get_fields() as $field_name => $field ) {
+				if ( ! hp\get_array_value( $field->get_args(), 'unique' ) || ! $field->get_value() ) {
```

Comment:
Once you move this option to the meta box, please use the prefixed name for it like "_unique", if I remember correctly we do this for "meta" options to avoid conflicts with possible field-level parameters set as field class variables.